installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -1,28 +1,34 @@
-[id='proc-install-rhdh-ocp-operator_{context}']
-
+[id="proc-install-rhdh-ocp-operator_{context}"]
 = Deploying {product} on {ocp-short} using the Operator
 
-As a developer, you can install {product} on your OpenShift Container Platform using the {product} Operator.
+As a developer, you can deploy a {product} instance on {ocp-short} by using the *Developer* perspective in the web console.
 
 .Prerequisites
-* Your administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
-* You have created a project in {ocp-short} for your {product} instance. For more information about creating a project in OpenShift, see link:https://docs.openshift.com/container-platform/4.14/applications/projects/working-with-projects.html#odc-creating-projects-using-developer-perspective_projects[Red Hat OpenShift documentation].
+
+* A cluster administrator has installed the {product} Operator. For more information, see xref:proc-install-operator_admin-rhdh[Installing the {product} Operator].
 
 .Procedure
-. From the *Developer* perspective on the {product-short} web console, click *+Add*.
+
+. From the *Developer* perspective on the {ocp-short} web console, click *+Add*.
+. Create a project in {ocp-short} for your {product} instance, or select an existing project.
++
+[TIP]
+====
+For more information about creating a project in {ocp-short}, see link:https://docs.openshift.com/container-platform/4.15/applications/projects/working-with-projects.html#creating-a-project-using-the-web-console_projects[Creating a project by using the web console] in the {ocp-brand-name} documentation.
+====
```

Comment:
Creating a project must happen before "click *+Add*".

Reply:
true but if you don't have one you'll be prompted.

Reply:
![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/40172997/fc679d5b-d21a-40d0-8b39-caed6015b833)

![image](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/assets/40172997/156276e7-8304-46a4-b3ab-e627078f4d7b)

This actually isn't true - you can click +Add and then select or create a project, which IMHO is actually a nicer workflow

Reply:
I changed it back regardless, but I disagree that this is the best order of steps. We can revisit another time, though.